Userland/Shell/Shell.h

```diff
@@ -332,9 +332,9 @@ class Shell : public Core::EventReceiver {
     bool was_interrupted { false };
     bool was_resized { false };
 
-    DeprecatedString cwd;
-    DeprecatedString username;
-    DeprecatedString home;
+    DeprecatedString cwd { "" };
+    DeprecatedString username { "" };
+    DeprecatedString home { "" };
```

Comment:
The commit description doesn't seem relevant to this commit?

Reply:
It is actually relevant; the problem is that when we can't access /etc/passwd we have no idea what the current user or their directory are. The code handles this case properly, but never sets these strings in that case, leaving them null. That leads to a crash later on.